### ISSUE TITLE
stacked_themes build for non-mobile platforms w/ statusBarColorBuilder throws error

### ISSUE THREAD

**Issue by calvintam236** [Stacked-Org/stacked#404]
Currently, if you set `statusBarColorBuilder` in the `ThemeBuilder` and try to build for non-Android and non-iOS platform, you will get this error:

```
[ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: MissingPluginException(No implementation found for method setstatusbarcolor on channel plugins.sameer.com/statusbar)
#0 MethodChannel._invokeMethod
package:flutter/…/services/platform_channel.dart:156
<asynchronous suspension>
#1 StatusBarService.updateStatusBarColor
package:stacked_themes/…/services/statusbar_service.dart:9
<asynchronous suspension>
#2 ThemeManager._applyStatusBarColor
package:stacked_themes/src/theme_manager.dart:161
<asynchronous suspension>
```

This should not throw error if the target platform does not support it.

**Comment by FilledStacks** [comment]
@hurbes you can implement for the platforms and simply do nothing to avoid the exception being thrown.

**Comment by FilledStacks** [comment]
fixed and merged. Will be released later today.

**Comment by FilledStacks** [comment]
Released in 0.3.4
